src/seccomp.c

```diff
@@ -17,8 +17,11 @@
 
 #include <packetgraph/common.h>
 #include <packetgraph/seccomp-bpf.h>
+#include <errno.h>
+#include <signal.h>
+#include <string.h>
 
-int pg_init_seccomp(void)
+int init_seccomp_filters(void)
```

Comment:
TBH, I don't understand the point of this sub function.
If you really want it please make it static.
```int init_seccomp_filters(void)```